mkosi/config.py

```diff
@@ -2575,8 +2579,14 @@ def finalize_defaults(namespace: argparse.Namespace, defaults: argparse.Namespac
 
         setting = SETTINGS_LOOKUP_BY_DEST[s]
 
-        die(f"{setting.long}={getattr(cli_ns, s)} was specified on the command line but is not allowed to be configured by any images.",
-            hint="Prefix the setting with '@' in the image configuration file to allow overriding it from the command line.")
+        die(
+            f"{setting.long}={getattr(cli_ns, s)} was specified on the command line "
+            "but is not allowed to be configured by any images.",
+            hint=(
+                "Prefix the setting with '@' in the image configuration file "
+                "to allow overriding it from the command line."
+            )
+        )
```

Comment:
No, please don't break messages like this arbitrarily. This makes grepping and udnerstanding what is going on harder.

Reply:
Yeah, I'm not entirely happy about this break, but to me those were already the closest semantic units. Without those breaks these lines are just way too long, though. 

I just pushed a differently terrible version. PTAL.